src/components/carsoulSection/carsoulSection.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect, useRef } from 'react';
+import {
+  MDBCarousel,
+  MDBCarouselInner,
+  MDBCol,
+  MDBRow,
+  MDBContainer,
+} from 'mdbreact';
+import './StylecarsoulSection/myStyle.scss';
+import CarouselItems from './CarsoureItem';
+import GrideCarousel from './GridCarousel';
+import HighLight from './HighlightSection';
+
+function CarsoulSection() {
+  const [activeCardItem, setActiveCardItemState] = useState(0);
+  const timerId = useRef();
+  useEffect(() => (timerId.current = resetTimer()), []);
+  let resetTimer = () => {
+    return setTimeout(() => handelChangeCard(true, 'unClicked'), 5000);
+  };
+  let handelChangeCard = (increase, buttom) => {
+    if (increase) {
+      buttom == 'unClicked'
+        ? setActiveCardItemState((oldCard) => (oldCard == 3 ? 0 : oldCard + 1))
```

Comment:
make the magic number 3 as a variable 
and I would love to see the cards object being mocked here since this is the place that manage there state